README.md

```diff
@@ -113,6 +113,7 @@ A collection of awesome useful and awesome links, resources and shiny things for
 * [Ency education](http://www.univ.ency-education.com/)
 vous trouverez sur cette plateform des cours et examens qui vous accompagnerons tout au long de votre cursus (Math ou Informatique )
 * [Université en ligne](http://uel.unisciel.fr/) Coherent selection of resources & Multimedia for students.
+* [MIT CSE Lectures](https://www.youtube.com/c/mitocw/playlists?view=50&sort=dd&shelf_id=5) Cours complets d'informatique du MIT.
```

Comment:
I think this better be a link to the Youtube channel (In the channels section).

Reply:
I have made the changes.